src/Infrastructure/Services/WebFileSystem.cs

```diff
@@ -50,13 +52,11 @@ private async Task<bool> UploadToWeb(string fileName, byte[] fileData, Cancellat
                 DataBase64 = Convert.ToBase64String(fileData),
                 FileName = fileName
             };
+
             var content = new StringContent(JsonSerializer.Serialize(request), Encoding.UTF8, "application/json");
 
-            using var message = await _httpClient.PostAsync(_url, content, cancellationToken);
-            if (!message.IsSuccessStatusCode)
-            {
-                return false;
-            }
+            // TODO: Write the actual File image upload logic to web.            
+            // Post this image binary content to an Image Upload API.
```

Comment:
```suggestion
            // TODO: Write the actual File image upload logic to web.            
            // Post this image binary content to an Image Upload API.
            await Task.CompletedTask;
```

We should await something to avoid the build warning.

Reply:
Removed unnecessary file `WebFileSystem.cs` and `IFileSystem`